src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -92,6 +92,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr import') as c:
         c.argument('source', help="The source identifier in the format '[registry.azurecr.io/]repository[:tag]' or '[registry.azurecr.io/]repository@digest'.")
         c.argument('source_registry', options_list=['--registry', '-r'], help='The source container registry can be name, login server or resource ID of the source registry.')
+        c.argument('source_registry_username', options_list=['--source-username', '-su'], help='The username of source container registry')
```

Comment:
Multi-character short options are not permitted.

Reply:
I think you can just use the existing username/password params rather than creating new ones, you might only need to update the help message since in this context they are for the source, rather than the target.